02_thermodynamics/README.md

```diff
@@ -0,0 +1,187 @@
+        3. The harmonic entropy, in ev/K/atom
+        4. The harmonic heat capacity in eV/K/atom
+    - The second one has been activated with the `--U0` options of `extract_forceconstants` and contains the $U_0$ correction to the harmonic free energy
+        1. The average potential energy, in eV/atom
+        2. The $U_0$ correction at second order, which is equal to `<V(R) - V_2(R)>`, in eV/atom.
+        3. The $U_0$ correction at third order, which is equal to `<V(R) - V_2(R) - V_3(R)>`, in eV/atom. Note that it will be equal to the second order correction if you didn't extract third order force constants
+        3. The $U_0$ correction at fourth order, which is equal to `<V(R) - V_2(R) - V_3(R) - V_4(R)>`, in eV/atom. Note that it will be equal to the second order correction if you didn't extract fourth order force constants
+    - To get the free energy with the second order correction, you just have to add the harmonic free energy and the second order correction in `outfile.U0` (the second value). 
+    The resulting free energy will be in eV/atom.
+
+- Try to converge the value of the harmonic free energy and the $U_0$ correction. For this, you have several parameters to control
+    1. The number of samples
+    2. The cutoff for the force constants
+    3. The q-point grid used to compute the harmonic properties. The grid can be controlled with the `--qg` option of the `phonon_dispersions` binary. For example, you can try
+    `phonon_dispersion_relation --dos --temperature 1300 --unit mev -qg 10 10 10`
+    - Each of these parameters will have a different influence on the free energy. For instance, the $U_0$ value is computed using the potential energy of each configuration. This means that contrary to the force constants, which benefit from $3 \times N_{\mathrm{at}}$ values per configurations, only one data point is added to the average per configurations. **Try to observe the effect of each of the parameter on the convergence of the $U_0$ and the harmonic free energy**. 
+
+Remember to rename the outfile.X before launching tdep again !
+
+## Practical example on the free energy convergence
+
+To grasp a better idea on how to converge the free energy, let's have a look at its computation from the start, using stochastic sampling.
+In the `reference/stochastic_sampling` folder, you will find everything needed to perform a self-consistent simulation of bcc Zr at 1100K at the same volume as before.
+If you need help on how to do so, don't hesitate to look back at the 01_sampling tutorial.
+
+When doing the iterations, look at the evolution of the harmonic free energy, $U_0$ correction term and the total free energy.
+Try to make the free energy converge to 1 meV/at.
+
+
+The steps to do the stochastic sampling are :
+1. Go to the folder `reference/stochastic_sampling/iter.000` to start the sampling
+2. Check the `Makefile` and the target `init`
+3. `make init` to create the first 4 samples
```

Comment:
**important**: do **not** start from scratch. Reuse force constants from last temperature. also do not necessarily start from step 0 but maybe 5 or so